rskj-core/src/main/java/org/ethereum/config/net/DevNetConfig.java

```diff
@@ -40,6 +43,7 @@
     public static DevNetConfig getDefaultDevNetConfig() {
         DevNetConfig config = new DevNetConfig();
         config.add(0, new DevNetOrchidConfig());
+        config.add(0, new DevNetSecondForkConfig());
```

Comment:
This line it's ok, but we should erase the previous configuration for devnet

Reply:
good catch, fixing it